src/modules/material-requests/dto/material-requests.dto.ts

```diff
@@ -39,6 +48,19 @@ export class MaterialRequestsQueryDto {
 	})
 	maintainerIds?: string[];
 
+	@IsBoolean()
+	@Type(() => Boolean)
+	@Transform((input) => {
+		return input.value === 'false' ? false : true;
+	})
+	@IsOptional()
+	@ApiPropertyOptional({
+		type: Boolean,
+		description: 'Is the material request pending or already requested',
+		default: false,
```

Comment:
wouldn't the default be null here, so we can omit the filter to not filter by isPending and return both pending and not pending material requests?
* isPending = false => only return sent material requests
* isPending = true => only return pending material requests
* isPending = undefined/null => return all material requests